.github/workflows/locale_po_to_json.yaml

```diff
@@ -0,0 +1,68 @@
+name: "Locale PO to JSON"
+
+on:
+  workflow_dispatch:
+permissions:
+  contents: read
+jobs:
+  locale_po_to_json:
+    if: github.repository_owner == 'ManageIQ'
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: manageiq/postgresql:13
+        env:
+          POSTGRESQL_USER: root
+          POSTGRESQL_PASSWORD: smartvm
+          POSTGRESQL_DATABASE: vmdb_i18n
+        options: --health-cmd pg_isready --health-interval 2s --health-timeout 5s --health-retries 5
+        ports:
+        - 5432:5432
+    env:
+      PGHOST: localhost
+      PGPASSWORD: smartvm
+      RAILS_ENV: i18n
+      SKIP_TEST_RESET: true
+      SKIP_DATABASE_SETUP: true
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up system
+      run: bin/before_install
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@v1
+      with:
+        ruby-version: "3.1"
```

Comment:
Worried we're going to forget this, but this is fine for now

Reply:
yeah, shared steps cross repo would be nice... it was ruby 3.0 in the core repo. I need to change that.